nevergrad/benchmark/plotting.py

```diff
@@ -28,6 +29,18 @@
 
 # %% Basic tools
 
+def _slugify(value: str) -> str:
+    """
+    Normalizes string, converts to lowercase, removes non-alpha characters,
+    and converts spaces to hyphens.
+    """
+    import unicodedata
+    value = unicodedata.normalize('NFKD', value).encode('ascii', 'ignore')  # type: ignore
+    value = unicode(re.sub('[^\w\s-]', '', value).strip().lower())  # type: ignore
+    value = unicode(re.sub('[-\s]+', '-', value))  # type: ignore
```

Comment:
Why does it need type ignore everywhere, what was the error? This is definitely not a good sign.

Reply:
Fixed.